aws/rust-runtime/aws-inlineable/Cargo.toml

```diff
@@ -19,9 +19,10 @@ http_1x = ["dep:http-1x", "dep:http-body-1x", "aws-smithy-runtime-api/http-1x"]
 # https://github.com/tkaitchuck/aHash/issues/200
 # when built with `cargo update -Z minimal-versions`
 ahash = { version = "0.8.11", default-features = false }
-aws-credential-types = { path = "../aws-credential-types" }
+aws-credential-types = { path = "../aws-credential-types", features = ["test-util"] }
```

Comment:
**fix**: We shouldn't enable `test-util` as part of `[dependencies]`

Reply:
oops